src/components/top-bar/brand-logo/app-logo.tsx

```diff
@@ -0,0 +1,85 @@
+import Reapit from './logos/reapit.svg?react'
+import ConsoleOwner from './logos/console-owner.svg?react'
+import ConsolePay from './logos/console-pay.svg?react'
+import ConsoleTenant from './logos/console-tenant.svg?react'
+import ReapitConnect from './logos/reapit-connect.svg?react'
+import ReapitProjector from './logos/reapit-projector.svg?react'
+import ReapitSales from './logos/reapit-sales.svg?react'
+import ReapitLettings from './logos/reapit-lettings.svg?react'
+import ReapitPM from './logos/reapit-pm.svg?react'
+import PMDemo from './logos/pm-demo.svg?react'
+import PMSales from './logos/pm-sales.svg?react'
+import PMInspect from './logos/pm-inspect.svg?react'
+import ReapitForms from './logos/reapit-forms.svg?react'
+import ReapitWebsites from './logos/reapit-websites.svg?react'
+import ReapitProposals from './logos/reapit-proposals.svg?react'
+import KeyWhere from './logos/key-where.svg?react'
+import AutoResponder from './logos/auto-responder.svg?react'
+
+export const supportedAppNames = [
```

Comment:
Suggestion: maybe a text enum might be slightly nicer here as you don't need to define SupportedAppName as a type something like:

```ts
export enum SupportedAppNames {
   consoleOwner = 'Console Owner',
   reapitConnect = 'Reapit Connect',
.....
}

// usage 

SupportedAppNames.reapitConnect // 'Reapit Connect'

```

Just a thought!